aprilgroup_tracking/calibration/calibrate_camera.py

```diff
@@ -0,0 +1,157 @@
+'''
+Camera Calibration
+
+Approximately > 15 images are taken of different angles and rotations of a chessboard.
+These images are used to calibrate the camera.
+If camera intrinsics already exists, it loads them, else it uses the images taken to initiate calibration.
+
+Important when taking images:
+1. Ensure that the chessboard is on a flat surface. You want to get the distortions caused by the camera, not the object.
+2. Take many pictures as necessary. 
+3. Take pictures with different angles to improve the quality of the calibration.
+4. Use an environment with good illumination.
+
+'''
+
+import numpy as np
+import cv2
+import os
+from pathlib import Path
+
+
+class Calibration:
+    INTRINSIC_PARAMETERS_FILE = "aprilgroup_tracking/calibration/CameraParams.npz"
+    IMAGES_FOLDER = "aprilgroup_tracking/calibration/images"
+
+
+    def __init__(self, logger):
```

Comment:
Nit: Let's follow this style guide for naming conventions: https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations.
It's particularly useful in telling at a glance what's internal and public facing within a class.